.github/workflows/main.yml

```diff
@@ -196,7 +196,7 @@ jobs:
       - name: 💾 Install (cargo-binstall) 💾
         run: |
           rustup override set stable
-          cargo binstall -V || cargo install cargo-binstall@0.23.1
+          cargo binstall -V || cargo install cargo-binstall@1.4.2
```

Comment:
cargo-binstall provides a github action that will install pre-built version of cargo-binstall

Reply:
I'll have to look that up, thanks.

Reply:
Updated to use action.